classes/ticket.js

```diff
@@ -0,0 +1,200 @@
+const { Collection } = require("discord.js");
+const Discord = require("discord.js");
+const discordServices = require('../discord-services');
+
+class Ticket {
+    constructor(guild, question, caveOptions, requester, hackers, number, ticketMsg, inactivePeriod, bufferTime) {
+        this.guild = guild;
+        this.category;
+        this.question = question;
+        this.text;
+        this.voice;
+        this.caveOptions = caveOptions;
+        this.requester = requester;
+        this.hackers = hackers;
+        this.hackercount = hackers.size + 1;
```

Comment:
duplicate information, basically hackers.size. I would just add all the hackers to that variable, to make it real. And then not have this variable. If you need this value a lot, you can create a function that will return the number of hackers.